src/Pages/category/components/CategoryContent/CategoryFilter.js

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+import { FILTER_DATA } from './mockData';
+
+function CategoryFilter() {
+    const [sortValue, setSortValue] = useState(FILTER_DATA[0].sorting);
+    const [isView, setIsView] = useState(false);
+
+    const onClickViewBtn = e => {
+        const { name } = e.target;
+        setIsView(!isView);
+        setSortValue(name);
+    };
+
+    const onChangeSortValue = e => {
+        const { value } = e.target;
+        setSortValue(value);
+    };
+
+    const totalProductCount = countNumber =>
+        '전체 ' + countNumber.toLocaleString() + '개';
+
+    return (
+        <CategoryFilterWrap>
+            <TotalProduct>{totalProductCount(400000)}</TotalProduct>
```

Comment:
```suggestion
    const totalProductCount = countNumber =>
        ;

    return (
        <CategoryFilterWrap>
            <TotalProduct>전체 {400_000.toLocaleString()}개</TotalProduct>
```
- 이 경우에는 jsx 속에 포함시켜주는 것이 위에서 아래로 시선이 흐르며 코드를 한번에 이해하기에 좋아보입니다.
- `_`를 쉼표 대신 숫자의 구분자로 사용할 수 있습니다. `400_000`이라고 사용하더라도 `400000`와 동일하게 처리됩니다.

Reply:
처음 구현했을 때 jsx안에 작성했었는데 다른 방법을 생각하다 시도해봤었습니다!ㅎㅎ
흐름에 따라 읽히는 것을 생각하면 수정 할 방법이 좋을 것 같습니다.
`_`는 처음 알게됐습니다 감사합니다~!